### ISSUE TITLE
LCD Pinout

### ISSUE THREAD

**Issue by spectrumm** [dekuNukem/Nintendo_Switch_Reverse_Engineering#71]
Has anyone reversed the lcd? I'm working on a custom project, and i bought a LCD, same as in the switch, to use with a raspberry, if possible id need the pinouts to connect it

**Comment by spectrumm** [comment]
I cant seem to find any HDMI to DSI boards?

**Comment by tpimh** [comment]
I have found this display: [E555HBM2](http://www.panelook.com/E555HBM2_EDO_5.5_OLED_overview_25401.html). Dimensions seem pretty similar to Switch original LCD, the connector is 30pin MIPI DSI and it is OLED. I wonder if it can be useful for your project.

**Comment by tpimh** [comment]
Oops, just learned that I was wrong about the dimensions. Switch uses 6.2" screen, not 5.5" (Switch Lite uses 5.5").

**Comment by ArcadeHustle** [comment]
They claim you can't trick Switch Lite into connecting to a TV.... I bet we can. https://www.cnet.com/news/you-cant-trick-nintendo-switch-lite-into-connecting-to-a-tv/ 

Perhaps need an FPGA to help, but I bet it can be done

**Comment by ArcadeHustle** [comment]
@spectrumm did you see this https://hackaday.com/2014/08/19/a-mipi-dsi-display-shieldhdmi-adapter/
![6175621407959806057](https://user-images.githubusercontent.com/41961787/66628458-e3d8c200-ebcc-11e9-95e5-970895ff435f.jpg)

**Comment by RetepV** [comment]
> @spectrumm did you see this https://hackaday.com/2014/08/19/a-mipi-dsi-display-shieldhdmi-adapter/
> ![6175621407959806057](https://user-images.githubusercontent.com/41961787/66628458-e3d8c200-ebcc-11e9-95e5-970895ff435f.jpg)

No need for an fpga. This thing can do that hands down and probably cheaper.

https://www.analog.com/media/en/technical-documentation/data-sheets/ADV7533.pdf

**Comment by andreacampanella** [comment]
No, that's a mipi to hdmi, it's used to add hdmi output to a host device with mipi, you need the other way around.

**Comment by RockoBravo** [comment]
There should be a solution that doesn't require additional power.  It should be possible to jump of the LCD pins. Has anyone found a pin out?

**Comment by maartenvb040476** [comment]
i am looking to make the lcd of the switch or switch lite work on the raspberry pi
so i made this schema and i know it can work like this but it is possible i am wrong about the data lines
https://easyeda.com/editor#id=85452c47e15d47ccb713e906936381b2
maybe someone with the right equipment kan recorded the signal from a switch

**Comment by spookeo** [comment]
Has anyone achieved this?

**Comment by tanNhungPhe** [comment]
anyone have information ???
